assets/airports/airportLoadList.js

```diff
@@ -72,6 +72,12 @@ window.AIRPORT_LOAD_LIST = (function() { // eslint-disable-line wrap-iife
             wip: true
         },
         {
+            icao: 'efhk',
+            level: 'easy',
```

Comment:
Ok, thanks. It was somewhat challenging to pick this as so far I've not found any criteria for selecting it.

Anyway, it was just me randomly picking between easy & medium that felt the most appropriate ones for me. I didn't feel too challenging after I did reduction to the current level of flights (and now you asked me below to make it even easier by reducing the number of flights even more ;-)).